quasar-core/src/main/java/co/paralleluniverse/fibers/instrument/MethodDatabase.java

```diff
@@ -515,17 +533,7 @@ private static boolean isJavaxInternal(String className) {
             return false;
         }
         final String classSubName = className.substring("javax/".length());
-        return classSubName.startsWith("activation/")
-                || classSubName.startsWith("crypto/")
-                || classSubName.startsWith("lang/")
-                || classSubName.startsWith("management/")
-                || classSubName.startsWith("naming/")
-                || classSubName.startsWith("net/")
-                || classSubName.startsWith("script/")
-                || classSubName.startsWith("security/")
-                || classSubName.startsWith("sql/")
-                || classSubName.startsWith("tools/")
-                || classSubName.startsWith("xml/");
+        return JDK_JAVAX_PACKAGES.stream().anyMatch(classSubName::startsWith);
```

Comment:
`JDK_JAVAX_PACKAGES` is missing "activation/"

Reply:
Thanks for spotting that. I queried the `javax.*` packages from the Java 11 JDK using the `jrt:/` filesystem, and then copied the list over to Java 8. I didn't realise that Java 11 had lost a few `javax.*` packages.